### PR TITLE
docs: first-cut guiding principles

### LINKED ISSUE #7: Principles of the operating guide
Create a set of guiding principles that the technical committee members would adhere to when formulating guidelines.

### PR DESCRIPTION
**Pull request by liangyuanruo**
## Description

Guiding principles for the operational guide, inspired by recent pull request discussions, along with references from @wjchen0623 first-cut Word document. To assist in its adoption, a Github pull request template has also been created.

As this goes beyond any specific engineering practice, I would like to seek wider inputs from the committee before merging this PR.

Fixes #7

### CODE REVIEW

File: guiding-principles.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Guiding principles
+
+This guide is a living document that is continually updated to support experimentation in software development practices.
+
+It does not aim to be all-encompassing; rather, the goal is to *serve* development teams by providing *relevant* and *salient* guidelines that raises the level of cybersecurity.
```
Comment (hunter) [inline guiding-principles.md:5]: Thoughts on also including engineering practice? eg.

```suggestion
It does not aim to be all-encompassing; rather, the goal is to *serve* development teams by providing *relevant* and *salient* guidelines that raises the level of cybersecurity and software engineering practices.
```

File: guiding-principles.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+| Darren Teo        | DyGCDTO, SNDGO          | Chairperson  |
+| Kevin Ng          | D/CODEX, GovTech        | GDS / CODE   |
+| Hunter Nield      | Distinguished Engineer  | GDS / CODE   |
+| Liang Yuanruo     | Engineering Manager     | OGP          |
+| Timothee Groleau  | Lead Software Engineer  | OGP          |
+| Chong Rong Hwa    | D/ACCS, GovTech         | CSG / GVG    |
+| Yeo Beng Huay     | D/GPP, GovTech          | CSG / GVG    |
+| Chen Weijun       | DD/SCQA, SNDGO          | Secretary    |
+
+Changes to the guide should be proposed by one of the groups, and **must** reviewed by an officer from a different group before being merged. In the event of deadlock, discussions and consensus can take place at regular Technical Committee meetings called by the Secretary. Learning points and key recommendations for mainstreaming into IM8 will also be tracked by the Secretary.
```
Comment (hunter) [inline guiding-principles.md:39]: Minor note that we might be able to encode this as CODEOWNER

**Comment by hunter** [comment]
I'm good to go. Do we want to change from a DRAFT so we can approve?

File: guiding-principles.md (hunk 2 above)
Comment (liangyuanruo) [reply guiding-principles.md:39]: Thanks for the tip! I think it's probably overkill for now, but eventually we might demarcate clearer areas of responsibility for review.

**Comment by wjchen0623** [comment]
Thanks @liangyuanruo for incorporating. Good with me too!

**Review by ronghwa80 — APPROVED** [review]
Thanks for this changes @liangyuanruo ! They looks good to me. :)

**Pull request merged**